lib/theme/colors.js

```diff
@@ -29,7 +29,7 @@ const defaultColors = {
     900: '#141414',
     800: '#313233',
     700: '#4E5052',
-    600: '#76777A',
+    600: '#757677',
```

Comment:
It's a small change, almost imperceptible by the eye, yet it makes texts with color="600" match the `4.5:1` contrast ratio required by W3C's standards